app/src/main/java/com/example/wangyf/android_dokumi_test/MainActivity.java

```diff
@@ -14,5 +21,28 @@ protected void onCreate(Bundle savedInstanceState) {
 
         TextView textView = (TextView)findViewById(R.id.test1);
         textView.setText("aaaaaaaaaaaaaaaaaaaa");
+
+
+        if (true){
+            Integer i = null;
+            i.toString();
+
+        }
+
+        SharedPreferences.Editor editor = PreferenceManager.getDefaultSharedPreferences(this).edit();
+        editor.commit();
+
+        method();
     }
+
+    public void method() {
+        String text = null;
+        System.out.println("text.length=" + text.length());
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/wang6793453/android-dokumi-test/blob/a6c0877d903d740a474dba9fb997d65f5693cfac/app/src/main/java/com/example/wangyf/android_dokumi_test/MainActivity.java#L40"></span>[SystemPrintln] System.out.println is used<br />
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/logging-java.html#SystemPrintln</td>
    </tr>
  </tbody>
</table>